GitHubComments.ps1

```diff
@@ -1,6 +1,8 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 
+$script:MediaTypeVersion = 'v3'
```

Comment:
Move this to `GitHubCore.ps1` (it will still be accessible from here)

Reply:
Prefer `Set-Variable -Scope Script ...` so that you can use other options like `ReadOnly`

Reply:
If your script starts with multiple variable declarations, you can use a hashtable to make the process more efficient:
```powershell
# Define script-scope, readonly variables
@{
   # myVar1 is useful for ...
   myVar1 = 'hello'

   # version is used for the API..
    requestVersion = 'v3'

}.GetEnumerator() | ForEach-Object {
    Set-Variable -Scope Script -Option ReadOnly -Name $_.Key -Value $_.Value
}
```